### PR TITLE
Improve add photo method

### PR DESCRIPTION
**Pull request by decobots**

### CODE REVIEW

File: src/data_base.py [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import logging
-from typing import List
+from typing import List, Dict
 from urllib import parse
 
 import psycopg2
 from psycopg2 import sql
```
Comment (livercat) [inline src/data_base.py]: Import sql, extras

File: src/data_base.py [hunk 2]
```diff
@@ -48,16 +50,16 @@ def create_table(self, table_name: str):
                                                     UNIQUE(id) )''').format(sql.Identifier(table_name)))
         log.debug(f"created (if not exist) table {table_name}")
 
-    def add_photo(self, id_photo: str):
+    def add_photos(self, photos: Dict[str, Photo]):
+        ids = [(photo,) for photo in photos]
```
Comment (livercat) [inline src/data_base.py]: ids = photos.keys()

File: tests/test_data_base.py [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 import psycopg2
 from psycopg2 import sql
+from psycopg2 import extensions
```
Comment (livercat) [inline tests/test_data_base.py]: Unused import?

File: src/data_base.py (hunk 1 above)
Comment (decobots) [reply src/data_base.py]: done

File: src/data_base.py (hunk 2 above)
Comment (decobots) [reply src/data_base.py]: all ok

File: tests/test_data_base.py (hunk 3 above)
Comment (decobots) [reply tests/test_data_base.py]: used in test_db_context_manager

**Pull request merged**